contracts/0.8.9/oracle/AccountingOracle.sol

```diff
@@ -683,6 +683,8 @@ contract AccountingOracle is BaseOracle {
 
     function _processExtraDataItems(bytes calldata data, ExtraDataIterState memory iter) internal {
         uint256 dataOffset = iter.dataOffset;
+        uint256 maxNodeOperatorsPerItem = 0;
+        uint256 maxNodeOperatorItemIndex = 0;
 
         /// @solidity memory-safe-assembly
```

Comment:
Not related to the changes, but I believe that it should be placed right above the `assembly {}` block according to the docs https://docs.soliditylang.org/en/latest/assembly.html#memory-safety

Reply:
I cannot see anything in the docs about memory safety that would imply this

Reply:
could you please point to a specific place?

Reply:
ahhhh, I missed that this comment is in wrong place. I thought you're talking about the two `uint256` variables above

Reply:
sorry